### PR TITLE
allow for multiple names per kind

### PR DESCRIPTION
**Pull request by steiler**
With this PR multiple kind names can be assigned to a kind implementation.

Kinds will register themselves with all their defined `kindnames`.
The default credentials map is now filled from the kind implementation via the `nodes.SetDefaultCredentials(...)` function.
Retrieving the default credentials works via `nodes.GetDefaultCredentialsForKind(<AnyOfTheAssignedKindNames>)`

I want to also get rid of the remaining:
```
const (
	NodeKindBridge          = "bridge"
	NodeKindHOST = "host"
	NodeKindOVS  = "ovs-bridge"
	NodeKindSRL  = "srl"
)
```
however, therefore we need to also work on the clab.config.go since it checks these Kinds. Therefore I had to also come up with a hack in the srl.init() where the kind is overwritten and forced to be nodes.NodeKindSRL.

### CODE REVIEW

**Comment by steiler** [comment]
issue for the kind consts created #845

**Comment by steiler** [comment]
Implementing #801

**Comment by hellt** [comment]
crazy week here. will try to review that by the mid next week

File: nodes/node.go [hunk 1]
```diff
@@ -23,35 +24,25 @@ const (
...
-	NodeKindVrXRV            = "vr-xrv"
-	NodeKindVrXRV9K          = "vr-xrv9k"
-	NodeKindVrNXOS           = "vr-nxos"
-	NodeKindIPInfusionOCNOS  = "ipinfusion-ocnos"
-	NodeKindKeysightIxiaCOne = "keysight_ixia-c-one"
+	NodeKindBridge = "bridge"
+
+	NodeKindHOST = "host"
+	NodeKindOVS  = "ovs-bridge"
+	NodeKindSRL  = "srl"
```
Comment (hellt) [inline nodes/node.go:31]: thanks @steiler this change LGTM!
I think we can chase those dependencies in config.go in a separate PR

**Review by hellt — APPROVED** [review]
